test/smoke/interior/README.md

```diff
@@ -0,0 +1,36 @@
+# Interior and Edge meshes
+
+Deploys two interior router meshes (running in same namespace, within the same
+context). Two edge routers will connect to each interior mesh.
```

Comment:
modify this info here to be accurate with this new scenario

Reply:
ok, Description updated